src/perform_step/low_order.jl

```diff
@@ -229,3 +229,127 @@ end
 
   end
 end
+
+@muladd function perform_step!(integrator,cache::RKMil_GeneralConstantCache,f=integrator.f)
+  @unpack t,dt,uprev,u,W,p = integrator
+  Wik = cache.WikJ
+  dW = W.dW
+
+  WikJ = get_iterated_I!(dW, Wik)
+
+  if alg_interpretation(integrator.alg) == :Ito
+    if typeof(dW) <: Number || is_diagonal_noise(integrator.sol.prob)
+      WikJ = WikJ .- 1//2 .* dt
+    else
+      WikJ -= 1//2 .* dt .* Eye{eltype(dW)}(length(dW))
+    end
+  end
+
+  du₁ = integrator.f(uprev,p,t)
+  L = integrator.g(uprev,p,t)
+  mil_correction = zero(u)
+
+  if typeof(dW) <: Number || is_diagonal_noise(integrator.sol.prob)
+    K = @.. uprev + dt*du₁
+    utilde = (alg_interpretation(integrator.alg) == :Ito ? K : uprev) + L*integrator.sqdt
+    ggprime = (integrator.g(utilde,p,t) .- L) ./ (integrator.sqdt)
+    mil_correction = ggprime .* WikJ
+    u = K + L .* dW + mil_correction
+  else
+    for i in 1:length(dW)
+      K = uprev + dt*du₁ + sqdt*@view(L[:,j])
+      gtmp = integrator.g(K,p,t)
+      ggprime = (gtmp - L)/sqdt
+      ggprime_norm = zero(eltype(u))
+      if integrator.opts.adaptive
+        ggprime_norm += integrator.opts.internalnorm(ggprime, t)
+      end
+      mil_correction += ggprime*@view(WikJ[:,j])
+    end
+    if integrator.opts.adaptive
+      K = @.. uprev + dt*du₁
+      u = K + L*dW + mil_correction
+    else
+      u = uprev + dt*du₁ + L*dW + mil_correction
+    end
+  end
+
+  if integrator.opts.adaptive
+    du₂ = integrator.f(K,p,t+dt)
+    if typeof(dW) <: Number || is_diagonal_noise(integrator.sol.prob)
+      tmp  = dt*(du₂ -du₁)/2
+      En = W.dW.^3 .* ((du2-L)/(integrator.sqdt)).^2 / 6
+      tmp = calculate_residuals(tmp, En, uprev, u, integrator.opts.abstol,
+            integrator.opts.reltol, integrator.opts.delta, integrator.opts.internalnorm, t)
+    else
+      En = integrator.opts.internalnorm(W.dW,t)^3*ggprime_norm^2 / 6
+      tmp = @.. integrator.opts.internalnorm(dt*(du₂ - du₁)/2,t)
+      tmp = @.. calculate_residuals(tmp, En, uprev, u, integrator.opts.abstol,
+                integrator.opts.reltol, integrator.opts.delta, integrator.opts.internalnorm, t)
+    end
+    integrator.EEst = integrator.opts.internalnorm(tmp, t)
+  end
+  integrator.u = u
+end
+
+@muladd function perform_step!(integrator,cache::RKMil_GeneralCache,f=integrator.f)
+  @unpack du₁, du₂, K, tmp, ggprime, L, mil_correction = cache
+  @unpack t,dt,uprev,u,W,p = integrator
+  dW = W.dW; sqdt = integrator.sqdt
+  Wik = cache.WikJ
+  get_iterated_I!(dW, Wik)
+  WikJ = Wik.WikJ
+
+  integrator.f(du₁,uprev,p,t)
+  integrator.g(L,uprev,p,t)
+  @.. mil_correction = zero(eltype(u))
+  ggprime_norm = zero(eltype(ggprime))
```

Comment:
just do `local ggprime_norm`

Reply:
I'm not sure if I'm getting what you are saying here.

Reply:
instead of initializing it you can just localize it